meta-iot2000-example/recipes-example/mraa/mraa_%.bbappend

```diff
@@ -13,10 +13,15 @@ RDEPENDS_${PN}-java = "java2-runtime"
 PROVIDES = "node-mraa"
 
 #The mraa project has some trouble installing the module in the right place, so here is an workaround.
-#PYTHON_SITEPACKAGES_DIR contains "dist-packages", I really don't know why...
+#PYTHON_SITEPACKAGES_DIR contains "dist-packages", propably the distro is detected as Debian-like
+#So we simply create symlinks here and skip the QA checks to avoid build errors
+INSANE_SKIP_${PN} += "dev-so"
+INSANE_SKIP_python-${PN} += "dev-so"
 
 FILES_python-${PN} = "${PYTHON_SITEPACKAGES_DIR}/../site-packages"
+
 do_install_append(){
 	install -d ${D}/usr/lib/python2.7/site-packages/
-	cp ${D}/usr/lib/python2.7/dist-packages/* ${D}/usr/lib/python2.7/site-packages
+	ln -s ../dist-packages/_mraa.so ${D}/usr/lib/python2.7/site-packages/
```

Comment:
As you're hard-referencing a particular Python version here (2.7), you should add a dependency on this very version into this `.bbappend` file for documentation purposes and to avoid strange build errors due to non-existing directories once Python (and/or the base recipe) is upgraded. Most likely, the `.so` won't work then, too, giving potentially more obscure failures.